### PR TITLE
workflow: Add tutorial for resharding workflow for local environment.

### PR DESCRIPTION
**Pull request by wangyipei01**

### CODE REVIEW

File: doc/HorizontalReshardingWorkflowGuide.md [hunk 1]
```diff
@@ -0,0 +1,203 @@
...
+    Initialize replication by electing the first master for each of the new shards:
+
+    ``` sh
+    vitess/examples/local$ ./lvtctl.sh InitShardMaster -force test_keyspace/-80 test-200
+    vitess/examples/local$ ./lvtctl.sh InitShardMaster -force test_keyspace/80- test-300
+    ```
+
+    After this set up, you should see the shards on Dashboard page of vtctld UI (http://localhost:15000). There should be 1 serving shard named    "0" and 2 non-serving shards named "-80" and "80-". Click the shard node, you can inspect all its tablets information.
+
+1.  Bring up a vtworker process, which can be conntected through port 15033. (The number of *vtworker* should be the same of
```
Comment (alainjobart) [inline doc/HorizontalReshardingWorkflowGuide.md]: typo: conntected

File: doc/HorizontalReshardingWorkflowGuide.md [hunk 2]
```diff
@@ -0,0 +1,203 @@
...
+    defines the type of workflow you want to create.
+ *  Fill in *test_keyspace* in the "Keyspace" slot.
+ *  Fill in *localhost:15033* in the "vtworker Addresses" slot.
+ *  Unselect the "enable_approvals" checkbox if you don't want to manually
+    approve task executions for canarying. (We suggest you to keep the default selected choice since this will enable the canary feature)
+1.  Click "Create" button at the bottom of the dialog. You will see a workflow
+    node created in the *Workflows* page if the creation succeeds. The workflow has started running now if "Skip Start" is
+    not selected.
+
+Another way to start the workflow is through the vtctlclient command, you can also visualize the workflow on vtctld UI *Workflows* section after executing the command:
```
Comment (alainjobart) [inline doc/HorizontalReshardingWorkflowGuide.md]: line too long: we usually try to keep the formatting <80 characters per line.

File: doc/HorizontalReshardingWorkflowGuide.md (hunk 1 above)
Comment (wangyipei01) [inline doc/HorizontalReshardingWorkflowGuide.md]: Done.

File: doc/HorizontalReshardingWorkflowGuide.md (hunk 2 above)
Comment (wangyipei01) [inline doc/HorizontalReshardingWorkflowGuide.md]: Done.

**Pull request merged**